public/stylesheets/boardEditor.css

```diff
@@ -100,3 +99,25 @@ body.dark #board_editor .spare.white {
+  top: 6px;
+  left: 6px;
+  height: 80%;
+  width: 80%;
```

Comment:
the mix of `6px` and `80%` looks a bit hacky. also the cursor icon is to big imo.